src/Cache/NullHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Cache;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ */
+final class NullHandler implements HandlerInterface
+{
+    public function getFile()
+    {
+    }
+
+    public function read()
```

Comment:
depending on https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1888#discussion_r63658286
probably yes

for sure we need common interface